xyz

```diff
@@ -65,6 +68,7 @@ dir="$(cd -P "$(dirname "$path")" && pwd)"
 branch=master
 increment=patch
 message_template='Version X.Y.Z'
+commit_flags=''
```

Comment:
I think I'd prefer a "Boolean", though I agree it's a bit messy:

```diff
-run "git commit$commit_flags --message '$message'"
+run "git commit$([[ $edit == true ]] && printf ' --edit') --message '$message'"
```

I'd like to preserve the correspondence between command-line arguments and local variables.